helpers/lbryApi.js

```diff
@@ -1,15 +1,27 @@
 var axios = require('axios');
+var db = require("../models");
```

Comment:
Writing all paths as relative from the root project folder rather than relative from the current file will make for later easier refactoring IMO.